tests/test_templates.py

```diff
@@ -37,6 +37,7 @@ def test_env(env):
         fp = gen.get_masked_flowproject(p)
         fp.import_from(origin=gen.ARCHIVE_DIR)
         jobs = fp.find_jobs(dict(environment=_env_name(env)))
+        # fp.generate_aggregates()
```

Comment:
since `fp` didn't have jobs in it hence after adding jobs we need to register aggregates

Reply:
I see what you mean. Given our discussion on the feature document. It makes sense to keep this function. We still need to register at `__init__`, but it will likely be necessary in the public API as well.

Reply:
Yes, I agree. We are doing it now in this implementation